project/settings.py

```
@@ -195,9 +189,33 @@
+
+# Email server configuration
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_HOST_USER = config('EMAIL_HOST_USER')
+EMAIL_HOST_PASSWORD = config('EMAIL_HOST_PASSWORD')
+EMAIL_PORT = 587
+EMAIL_USE_TLS = True
+DEFAULT_FROM_EMAIL = config('DEFAULT_FROM_EMAIL')
+
+
+JITSI_APP_ID = "vpaas-magic-cookie-fb583aeece2040b4805f50d0b267e75b"
+JITSI_SECRET = "vpaas-magic-cookie-fb583aeece2040b4805f50d0b267e75b/ee88ba"
```

Ruff (0.8.2)

Possible hardcoded password assigned to: "JITSI_SECRET"

(S105)